### PR TITLE
add axis handle in order to support subplot functionality; add freque…

### PR DESCRIPTION
**Pull request by awollbrink**
…ncy axis log scaling

### CODE REVIEW

**Review by schoffelen — COMMENTED** [review]
Hi Andreas, thanks for the changes. I am not a user of the function myself, but read through the changes, and have a few questions for clarification.

File: contrib/misc/ft_icabrowser.m [hunk 1]
```diff
@@ -1,4 +1,4 @@
-function [rej_comp] = ft_icabrowser(cfg, comp)
+function [rej_comp] = aw_ft_icabrowser(cfg, comp)
```
[the remark this answers is not in the document]
Comment (awollbrink) [reply contrib/misc/ft_icabrowser.m:1]: yes, sorry for forgetting about this.

File: contrib/misc/ft_icabrowser.m [hunk 2]
```diff
@@ -65,6 +67,10 @@
   powscale = 'log10';
 end
 
+if ~ismember(freqscale, {'lin', 'log'})
```
[the remark this answers is not in the document]
Comment (awollbrink) [reply contrib/misc/ft_icabrowser.m:70]: again my fault: line 71 should be 'freqscale = 'lin';'

File: contrib/misc/ft_icabrowser.m [hunk 3]
```diff
@@ -133,14 +139,25 @@
...
+      powlog.String = 'Log Power Scale';
+    end
+
+    % ------------------------------------------------
+    % SWITCH FREQUENCY AXIS LOG/LINEAR
+    % ------------------------------------------------
+    freqlog = uicontrol('Units','normalized','Position',[0.55 0.01 0.090 0.05],'Style','pushbutton', 'Callback',@(h, evt)freqplotlog);
+    freqlog.Enable = 'off';
+    if strcmp(freqscale, 'log')
+      freqlog.String = 'Linear Frequency Scale';
```
[the remark this answers is not in the document]
Comment (awollbrink) [reply contrib/misc/ft_icabrowser.m:158]: here my coding should be correct. powlog.String and freqlog.String are button labels not representing the current but the state after activating the button

[the remark this answers is not in the document]
Comment (schoffelen) [reply contrib/misc/ft_icabrowser.m:158]: a good point.

**Review by schoffelen — APPROVED** [review]

**Comment by schoffelen** [comment]
Thanks Andreas!

**Pull request merged**